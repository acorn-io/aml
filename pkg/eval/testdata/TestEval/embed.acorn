a: 1
{
    a: 1
    d: 3
}
c: a
