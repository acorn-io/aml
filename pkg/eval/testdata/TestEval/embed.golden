{
  "a": 1,
  "c": 1,
  "d": 3
}